### ISSUE TITLE
Dice Score is low

### ISSUE THREAD

**Issue by zzh1230** [ge-xing/Diff-UNet#20]
I trained on the LiTS data set, and the Dice Score on the training set can reach 0.9035, but only 0.6898 on the test set. When I visualized the results, I found an interesting thing. The predicted shapes are very similar, but there seems to be some Offset, where might the problem be?
![image](https://github.com/ge-xing/Diff-UNet/assets/39183186/ac6c32f7-13a9-456e-9e7f-a3e6f22e3a4f)

**Comment by 920232796** [comment]
I do not encounter your problem before, it is very interesting. You can check if there are problems about resample or crop non-zero operation in the data loading processing.

**Comment by OCEANOUXIN** [comment]
Oh, I meet the same problem with you!  T^T
